sdk/opendp/smartnoise/sql/dpsu.py

```diff
@@ -6,7 +6,7 @@
 
 from opendp.smartnoise.sql.parse import QueryParser
 from opendp.smartnoise._ast.ast import Table
-from opendp.smartnoise.mechanisms.rand import laplace
+from opendp.smartnoise.sql._mechanisms.rand import laplace
```

Comment:
this can also use the ._mechanisms import but is mostly a nit

Reply:
Fixed